hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/LambdaTestUtils.java

```diff
@@ -382,7 +383,7 @@ public static <T, E extends Throwable> E intercept(
       Callable<T> eval)
       throws Exception {
     return intercept(clazz,
-        null,
+        (String) null,
```

Comment:
ooh, I worry if there are going to be link problems by anything not in our codebase using the method (gcs etc). Hope not.